pkg/manifest/operations.go

```diff
@@ -297,6 +303,10 @@ func (o *Operations) uninstall(deployInfo types.InstallInfo) (bool, error) {
 	return true, err
 }
 
+func noResourceFound(err error) bool {
+	return err == nil || apierrors.IsNotFound(err) || meta.IsNoMatchError(err)
```

Comment:
why is err==nil equivalent to no resource found? From what I understand this logic does not seem correct.

Reply:
Actually I add this just because of to prevent null point exception, but your are right, doesn't need to check it, it will handle inside those functions.